Source/WebKit/UIProcess/ProcessThrottler.cpp

```diff
@@ -394,6 +394,8 @@ void ProcessThrottler::sendPrepareToSuspendIPC(IsSuspensionImminent isSuspension
     } else {
         m_pendingRequestToSuspendID = generatePrepareToSuspendRequestID();
         Ref process = m_process.get();
+        if (process->isBeingDeleted())
```

Comment:
Have you considered relying on the `m_isConnectedToProcess` flag instead?

Reply:
That is a good point, will fix!

Thanks for reviewing!